Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -623,10 +623,10 @@ static WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     m_backgroundWebView.get().inspectable = inspectable;
 
     for (auto entry : m_extensionPageTabMap)
-        entry.key.cocoaView().get().inspectable = inspectable;
+        Ref { entry.key }->cocoaView().get().inspectable = inspectable;
 
     for (auto entry : m_popupPageActionMap)
-        entry.key.cocoaView().get().inspectable = inspectable;
+        Ref { entry.key }->cocoaView().get().inspectable = inspectable;
```

Comment:
Ditto on using a local var.

Reply:
You're right, it does look better with the local variable--fixed!